scripts/scil_merge_sh.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import argparse
+
+import nibabel as nb
+import numpy as np
+
+from scilpy.io.image import assert_same_resolution
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_outputs_exist)
+
+"""
+Merge 2 Spherical Harmonics files.
```

Comment:
Don't know if it makes sense but I think it would be interesting to modify this script to be able to merge multiple sh files together.

Reply:
I'm not sure about that but I can